packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -972,6 +972,11 @@ export interface ModelChangedEvent {
     readonly versionId: number;
 }
 
+export interface EnvVariable {
```

Comment:
`EnvVariableMain`, we have already `EnvVariable` in core

Reply:
Thx, will reuse interface from core.